### PR TITLE
build: add postcss loader

### PR DESCRIPTION
**Pull request by rishichawda**
Added postcss-loader to project - issue #95

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -57,13 +56,16 @@
     "html-webpack-plugin": "^3.2.0",
     "markdown-loader": "^0.1.7",
     "node-sass": "^3.7.0",
-    "postcss-loader": "^2.1.5",
+    "postcss-import": "^12.0.0",
```
Comment (rishichawda) [inline docs/package.json:59]: postcss-import plugin will be required only if we decide not to use css-loader.

**Comment by rishichawda** [comment]
@prasanna1211 Another advantage ( by what I understood ) about not using css-loader ( since now postcss-loader is used ) is that we won't need to use `@import` statements in scss files.

Should we completely remove css-loader?

**Comment by prasanna1211** [comment]
@rishichawda will look at it and let you know. Trying POST CSS for first time.

**Comment by prasanna1211** [comment]
@rishichawda I guess we need to drop this idea. The main reason for choosing POSTCSS was that it gives cross browser compatibility. But now I realize even SASS can do this. #86 will solve this.

**Comment by prasanna1211** [comment]
Something like https://stackoverflow.com/questions/19351530/cross-browser-mixin-using-sass

**Comment by rishichawda** [comment]
> @rishichawda I guess we need to drop this idea. The main reason for choosing POSTCSS was that it gives cross browser compatibility. But now I realize even SASS can do this. #86 will solve this.

Yes. Only advantage was that we could automate a lot of things using postcss and reduce how much code we need to write for each component.

**Comment by rishichawda** [comment]
> Something like https://stackoverflow.com/questions/19351530/cross-browser-mixin-using-sass

This looks similar to what I did for flex.

**Pull request closed without merging**